fraqbot/Local/shakespeare.py

```diff
@@ -1,49 +1,59 @@
-from Legobot.Lego import Lego
 import logging
 import os
 import random
 import sys
 
+from Legobot.Lego import Lego
```

Comment:
logging, os, random, sys are "standard libs"
Legobot is third party, so I split them, per the style guide.